BuildFile.xml

```diff
@@ -13,3 +13,4 @@
 <export>
     <lib name="1"/>
 </export>
+<flags CXXFLAGS="-g" />
```

Comment:
It does not hurt the runtime, but does provide line number after a crash for I'd keep it :+1: